include/zephyr/bluetooth/bluetooth.h

```diff
@@ -1096,6 +1096,13 @@ struct bt_le_ext_adv_info {
 	int8_t                     tx_power;
 };
 
+/**
+ * @brief Reset advertising flags.
+ *
+ * Clear all advertising flags.
+ */
+void bt_adv_reset_adv_flags(void);
```

Comment:
Don't put internal functions in the public API headers in `/include/zephyr/bluetooth`.